coprocess_python.go

```diff
@@ -187,6 +185,9 @@ const CoProcessName = apidef.PythonDriver
 // MessageType sets the default message type.
 var MessageType = coprocess.ProtobufMessage
 
+// pythonInitDone is used to signal the end of Python initialization step
+var pythonInitDone = make(chan bool)
```

Comment:
Is there a reason to make this a global?

If this is to reuse the channel allocation, are we sure it won't be used more than once concurrently?